cinder/README.md

```diff
@@ -27,20 +27,22 @@ The formal Dell EMC container image is published to [Red Hat Container Catalog](
 
 Red Hat OpenStack Platform supports remote registry and local registry for overcloud deployment. In this document, we only introduce local registry.
 
-> in below examples, 192.168.139.1:8787 acts as a local registry.
+> In below examples, 192.168.139.1:8787 acts as a local registry.
+
+**Notes:** We will not introduce how to setup local registry with docker-registry or docker-distribution. In RHOSP16,  undercloud also acts as a local registry, you could push Dell EMC container images to it, and pull these images when deploying overcloud, please refer Red Hat documents.
 
 Frist, login registry.connect.redhat.com and pull the container image from Red Hat Container Catalog.
 
 ```bash
 $ docker login -u username -p password registry.connect.redhat.com
-$ docker pull registry.connect.redhat.com/dellemc/openstack-cinder-volume-dellemc
+$ docker pull registry.connect.redhat.com/dellemc/openstack-cinder-volume-dellemc-rhosp16
 ```
 
 Then, tag and push it to the local registry.
 
 ```bash
-$ docker tag registry.connect.redhat.com/dellemc/openstack-cinder-volume-dellemc 192.168.139.1:8787/dellemc/openstack-cinder-volume-dellemc
-$ docker push 192.168.139.1:8787/dellemc/openstack-cinder-volume-dellemc
+$ docker tag registry.connect.redhat.com/dellemc/openstack-cinder-volume-dellemc-rhosp16  192.168.139.1:8787/dellemc/openstack-cinder-volume-dellemc-rhosp16
+$ docker push 192.168.139.1:8787/dellemc/openstack-cinder-volume-dellemc-rhosp16
 ```
 
 #### Prepare custom environment yaml
```

Comment:
Better to add a sequence number here like `#### 2. Prepare custom enviroment yaml` although you already give it a Header style.

Reply:
Good suggestion. Updated.